docs/events/mouse_click.md

```diff
@@ -0,0 +1,20 @@
+# Click
+
+The `Click` event is sent to a widget when the user clicks a mouse button.
+
+- [x] Bubbles
+
+## Attributes
```

Comment:
Other mouse events have the exact same table, so whatever changes are accepted here, should be copied and pasted over to `MouseDown`, `MouseMove`, and `MouseUp`.